wpeverest/everest-forms
# Fix - Form notices for frontend

### All Submissions:






### Changes proposed in this Pull Request:

This PR aims to solve notices issue within the frontend. Now it does displays the header error notice if any of the form fields have errors plus minimizes the use of `evf_add_notice` call which uses a session to display a notice.

### How to test the changes in this Pull Request:

1. Make a form submission or have an addon to test just submit.
2. If any error occurs, notice will appear on the header.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Show form error notice on any form field error.
